draft-birkholz-rats-coswid-rim.md

```diff
@@ -119,8 +119,8 @@ The design of the following attributes is motivated by the vast variety of ident
 | platform-manufacturer-name | 0-1 | An identifier number (uint) value that uniquely represents the firmware manufacturer. This identifier MUST be included unless the firmware manufacturer and the platform manufacturer are represented via the same number (unit) value, this attribute MAY be omitted.
 | platform-model-name | 1 | An identifier text (tstr) value enabling the identification of a certain device model/type composite. The reliability of this identifier is not absolute. In consequence this identifier MUST NOT be omitted. In an case, the use of this identifier requires foresight and preparation as it's purpose supports semantic interoperability. Arbitrary, conflicting, or unresolvable values SHOULD be avoided.
 | platform-version | 0-1 | A byte-comparable reference to a Platform Certificate's 'Manufacturer-Specific Identifier' extension value [ref TCG Platform Certificate Profile, Version 1.1 Revision 1 5 13 Feb 2019].
-| firmware-manufacturer-id | 1 | An IANA defined unique value that is a Private Enterprise Number (Platform manufacturer unique identifier) that MUST be included in a CoSWID RIM.
-| firmware-manufacturer-name | 0-1 | An identifier that is represented as the name of a platform manufacturer via a text (tstr) value that SHOULD be included in a CoSWID RIM.
+| firmware-manufacturer-id | 0-1 | An IANA defined unique value that is a Private Enterprise Number (Platform manufacturer unique identifier) that SHOULD be included in a CoSWID RIM that covers firmware.
```

Comment:
There might be consequences beyond we know at -00, but that is fine. Relationship with SUIT is also still open for exploration.